helper_test.go

```diff
@@ -11,3 +11,11 @@ func NewWithMockedTerminal() (*finder, *TerminalMock) {
 	m.SetSize(w, h)
 	return f, m
 }
+
+func NewWithMockedTerminalV2() (*finder, *TerminalMock) {
```

Comment:
There is no need to define a v2 function because `helper_test.go` is only for testing and it is referenced only from `fuzzyfinder_test.go`. So you can rename it to `NewWithMockedTerminal` and remove the old one.

```suggestion
func NewWithMockedTerminal() (*finder, *TerminalMock) {
```